broker/submissions/export_to_spreadsheet_service.py

```diff
@@ -46,11 +46,28 @@ def export_and_save(self, submission_uuid: str, storage_dir: str):
         os.makedirs(spreadsheet_details.directory, exist_ok=True)
         workbook.save(spreadsheet_details.filepath)
 
+        # dcp-845 - spreadsheet in staging area
+        # create a file payload that adhere to the schema
+        # https://schema.humancellatlas.org/type/file/2.5.0/supplementary_file
+        spreadsheet_payload = self.build_supplementary_file_payload(spreadsheet_details)
+        # POST /submission/<id>/files
+        self.ingest_api.put(f'/submission/{submission_uuid}/files/', spreadsheet_payload)
+        # payload as body
         finished_date = datetime.now(timezone.utc)
         self._patch(submission_url, create_date, finished_date)
 
         self.logger.info(f'Done exporting spreadsheet for submission {submission_uuid}!')
 
+    def build_supplementary_file_payload(self, spreadsheet_details):
+        return {
+            "describedBy": "https://schema.humancellatlas.org/type/file/2.5.0/supplementary_file",
+            "schema_type": "file",
+            "file_core": {
+                "file_name": spreadsheet_details.filename,
+                "format": "xlsx"
+            }
```

Comment:
According to the [dcp spec](https://github.com/HumanCellAtlas/dcp2/blob/e4d843f75ec66a2d5ef1dadad0c259703b1bb085/docs/dcp2_system_design.rst#ingest-spreadsheets) the the spreadsheet should be have the metadata that describes its source.
For that the payload should include the `file_source` and `content_description` attributes. The latter can have an ontology reference, which will be [`data:2193`](https://ontology.archive.data.humancellatlas.org/ontologies/edam/terms?iri=http%3A%2F%2Fedamontology.org%2Fdata_2193) as [decided previously](https://app.zenhub.com/workspaces/dcp-ingest-product-development-5f71ca62a3cb47326bdc1b5c/issues/ebi-ait/dcp-ingest-central/845#issuecomment-1249230627).

```suggestion
            "file_core": {
                "file_name": spreadsheet_details.filename,
                "format": "xlsx"
                "file_source": "DCP/2 Ingest",
                "content_description": {
                    "text":"metadata spreadsheet",
                    "ontology":"data:2193",
                }
            }
```

@ESapenaVentura please review the `text` I used for the `content_description`.

Reply:
```suggestion
            "file_core": {
                "file_name": spreadsheet_details.filename,
                "format": "xlsx",
                "file_source": "DCP/2 Ingest",
                "content_description": {
                    "text":"metadata spreadsheet",
                    "ontology":"data:2193",
                    "ontology_label":"Database entry metadata"
                }
            }
```

Reply:
@amnonkhen I would suggest this, it was missing the ontology_label. Other than that LGTM